### PR TITLE
New package: kiten-19.12.3

### PR DESCRIPTION
**Pull request by likrtr**

### CODE REVIEW

**Comment by Chocimier** [comment]
When fixed, please [fixup](https://git-scm.com/docs/git-rebase#_interactive_mode) commits into one.

**Comment by abenson** [comment]
Any movement on getting this fixed?

File: srcpkgs/kiten/template [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+homepage="https://kde.org/applications/education/org.kde.kiten"
+distfiles="${KDE_SITE}/release-service/${version}/src/${pkgname}-${version}.tar.xz"
+checksum=c5d002ba2ee4a11913457808da67f9c7ee5dbcf1731fad11842a4aa338862c82
+
+pre_check() {
+    export QT_QPA_PLATFORM=offscreen
+}
+
+if [ "$CROSS_BUILD" ]; then
+	hostmakedepends+="kcompletion kcompletion-devel kconfig kconfig-devel kconfwidgets-devel
```
Comment (ahesford) [inline srcpkgs/kiten/template:23]: Missing space before `kcompletion` corrupts `hostmakedepends`

Comment (likrtr) [reply srcpkgs/kiten/template]: Fixed in the latest commit

File: srcpkgs/kiten/template [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+depends="khtml hicolor-icon-theme"
+short_desc="Japanese Reference/Study Tool"
+maintainer="basketdou <basketdou@gmail.com>"
+license="GPL-2.0-or-later"
+homepage="https://kde.org/applications/education/org.kde.kiten"
+distfiles="${KDE_SITE}/release-service/${version}/src/${pkgname}-${version}.tar.xz"
+checksum=663739a8b252cb95a38294c6f7d675c833daaa81f2654f5cabd8e512fd353560
+
+if [ "$CROSS_BUILD" ]; then
+	hostmakedepends+=" kdoctools python qt5-host-tools qt5-qmake"
```
[the remark this answers is not in the document]
Comment (likrtr) [reply srcpkgs/kiten/template]: Fixed in the latest commit.

File: srcpkgs/kiten/template [hunk 3]
```diff
@@ -0,0 +1,19 @@
+# Template file for 'kiten'
+pkgname=kiten
+version=19.12.3
+revision=1
+build_style=cmake
+configure_args="-DBUILD_TESTING=OFF"
```
[the remark this answers is not in the document]
Comment (likrtr) [reply srcpkgs/kiten/template:6]: Fixed in the latest commit

[the remark this answers is not in the document]
Comment (Chocimier) [reply srcpkgs/kiten/template:6]: ok, please remove configure_args now.

File: srcpkgs/kiten/template [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+homepage="https://kde.org/applications/education/org.kde.kiten"
+distfiles="${KDE_SITE}/release-service/${version}/src/${pkgname}-${version}.tar.xz"
+checksum=c5d002ba2ee4a11913457808da67f9c7ee5dbcf1731fad11842a4aa338862c82
+
+pre_check() {
+export QT_QPA_PLATFORM=offscreen
+}
+
+if [ "$CROSS_BUILD" ]; then
+	hostmakedepends+=" kcompletion kcompletion-devel kconfig kconfig-devel kconfwidgets-devel
```
Comment (Chocimier) [inline srcpkgs/kiten/template:23]: *-devel depends on  *, no need to specify it separately.

Comment (Chocimier) [reply srcpkgs/kiten/template:23]: also *-devel should not be put into hostmakedepends at all, only makedepends

**Comment by likrtr** [comment]
Well, it seems that I'll have to remove this pull request and start from scratch in order to fix all the errors and merge in one commit.

**Pull request closed without merging**